pkg/controller/azure/provider/resourcegroup_test.go

```diff
@@ -0,0 +1,690 @@
+/*
+Copyright 2018 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package provider
+
+import (
+	"context"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/Azure/go-autorest/autorest"
+	"github.com/pkg/errors"
+
```

Comment:
I don't think this whitespace is needed - `goimports` tends to keep any whitespace that accidentally makes its way in. We aim to format the import clause as:

* stdlib packages
* One blank line
* Third party packages
* One blank line
* Crossplane packages